x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_page/alerts_page.tsx

```diff
@@ -135,29 +136,9 @@ const AlertsPage: React.FunctionComponent = () => {
   };
 
   const tableProps = {
+    pluginId: PLUGIN_ID,
```

Comment:
Hmm. Is this too limiting? Is it possible for a single plugin to potentially configure two separate alert tables? But if we don't use plugin id, we need to handle potential collisions (assuming we just allow arbitrary ids) - this may be fine for now and we can reassess if the need comes up in the future? WDYT?

Reply:
In reality this is not a plugin id, this is just an id. When you render the AlertsTable component you pass this id and it can be anything as long as it is registered.

Maybe we just call this `id` and for starters plugins can start using their pluginId on it but it is not limited as such? Maybe we can call id `tableId` or `configurationId` ?

Reply:
I like `configurationId` because it maps to the configuration registry